packages/contracts-bedrock/scripts/DeployOwnership.s.sol

```diff
@@ -0,0 +1,164 @@
+
+        save("LivenessGuard", address(addr_));
+        console.log("New LivenessGuard deployed at %s", address(addr_));
+    }
+
+    /// @notice Deploy a LivenessModule for use on the Security Council Safe
+    ///         Note this function does not have the broadcast modifier.
+    function deployLivenessModule() public returns (address addr_) {
+        Safe councilSafe = Safe(payable(mustGetAddress("SecurityCouncilSafe")));
+        address guard = mustGetAddress("LivenessGuard");
+        SecurityCouncilConfig memory councilConfig = _getExampleCouncilConfig();
+
+        addr_ = address(
+            new LivenessModule({
+                _safe: councilSafe,
+                _livenessGuard: LivenessGuard(guard),
+                _livenessInterval: councilConfig.livenessInterval,
+                _thresholdPercentage: councilConfig.thresholdPercentage,
+                _minOwners: councilConfig.minOwners,
+                _fallbackOwner: councilConfig.fallbackOwner
+            })
+        );
+
+        save("LivenessModule", address(addr_));
+        console.log("New LivenessModule deployed at %s", address(addr_));
+    }
+
+    /// @notice Deploy a Security Council with LivenessModule and LivenessGuard.
+    function deployAndConfigueSecurityCouncilSafe() public returns (address addr_) {
+        Safe safe = Safe(payable(deploySafe("SecurityCouncilSafe")));
+
+        address guard = deployLivenessGuard();
+
+        vm.startBroadcast();
+        _callViaSafe({ _safe: safe, _target: address(safe), _data: abi.encodeCall(GuardManager.setGuard, (guard)) });
+        console.log("LivenessGuard setup on SecurityCouncilSafe");
+
+        SecurityCouncilConfig memory exampleCouncilConfig = _getExampleCouncilConfig();
+        // Add the owners, keeping the threshold at 1 for now.
+        for (uint256 i = 0; i < exampleCouncilConfig.safeConfig.owners.length; i++) {
+            _callViaSafe({
+                _safe: safe,
+                _target: address(safe),
+                _data: abi.encodeCall(OwnerManager.addOwnerWithThreshold, (exampleCouncilConfig.safeConfig.owners[i], 1))
+            });
+        }
+        // Remove the deployer address which was used to setup the Security Council Safe thus far
+        // this call is also used to update the threshold.
+        _callViaSafe({
+            _safe: safe,
+            _target: address(safe),
+            _data: abi.encodeCall(
+                OwnerManager.removeOwner,
+                (exampleCouncilConfig.safeConfig.owners[0], msg.sender, exampleCouncilConfig.safeConfig.threshold)
+            )
+        });
+
+        address livenessModule = deployLivenessModule();
+        vm.stopBroadcast();
+
+        // Since we don't have private keys for the safe owners, we instead use 'startBroadcast' to do something
+        // similar to pranking as the safe. This simulates a quorum of signers executing a transation from the safe to
+        // call it's own 'enableModule' method.
```

Comment:
I think we should more clearly state somewhere in this file that this script / `deployAndConfigureSecurityCouncilSafe` is not actually executable because of this. 
```suggestion
        // Since we don't have private keys for the safe owners, we instead use 'startBroadcast' to do something
        // similar to pranking as the safe. This simulates a quorum of signers executing a transaction from the safe to
        // call it's own 'enableModule' method.
```

Reply:
The contract level natspec now says: 

```
/// @notice Script used to deploy and configure the Safe contracts which are used to manage the Superchain,
///         as the ProxyAdminOwner and other roles in the system. Note that this script is not executable in a
///         production environment as some steps depend on having a quorum of signers available. This script is meant to
///         be used as an example to guide the setup and configuration of the Safe contracts.
```